chore(rn): Share integrations documentation with javascript (#7009)
getsentry/sentry-docs: src/platforms/react-native/configuration/integrations/custom.mdx

---
title: Custom Integrations
sidebar_order: 3

> description: "Learn how you can enable a custom integration."

Reviewer: ```suggestion
description: "Learn how to enable a custom integration."
```
